Source/IntegrationTests/TestFiles/LitTests/LitTest/dafny4/Lucas-up.dfy

```diff
@@ -48,14 +49,20 @@ lemma BitSize(i: nat, n: nat)
 
 // An easy-to-read name for the expression that checks if a number
 // is even.
-ghost predicate EVEN(n: nat)
+opaque ghost predicate EVEN(n: nat)
```

Comment:
Again, try to avoid introducing `opaque function` in our tests

Reply:
Why?

Reply:
Because `opaque functions` are more cumbersome to use then regular ones. They need to be revealed in any situations where you want the body to be visible, and are hidden in situations even where no reduction in resource count is needed.

You can see the code size advantage of avoiding opaque functions in the refactoring in this PR: https://github.com/dafny-lang/dafny/pull/5781